.gitignore

```diff
@@ -7,6 +7,8 @@ upload-files/
 demo*
 gen-java
 *.class
+apache-hugegraph-incubating-1.0.0/
```

Comment:
it's redundent with line 61
<img width="349" alt="image" src="https://user-images.githubusercontent.com/17706099/209971376-e99a3db9-1f23-41d8-96ed-26989e402364.png">